### PR TITLE
Fix minor bugs regarding Label names & Scipy - YOLOv4

### PR DESCRIPTION
**Pull request by mustafakasap**
Line 78 changed from import scipy to  **from scipy import special** Before "special" was not imported by default

Line 216 changed from
     def draw_bbox(image, bboxes, classes=read_class_names(cfg.YOLO.CLASSES), show_label=True):
to 
     def draw_bbox(image, bboxes, classes=read_class_names("coco.names"), show_label=True):
as label names are not stored in cfg object as in tensorflow or darknet original samples.

### CODE REVIEW

**Comment by vinitra** [comment]
Can you make the changes from #350 in this PR itself?

File: vision/object_detection_segmentation/yolov4/README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ def postprocess_bbbox(pred_bbox, ANCHORS, STRIDES, XYSCALE=[1,1,1]):
         xy_grid = np.tile(np.expand_dims(xy_grid, axis=0), [1, 1, 1, 3, 1])
         xy_grid = xy_grid.astype(np.float)
 
-        pred_xy = ((scipy.special.expit(conv_raw_dxdy) * XYSCALE[i]) - 0.5 * (XYSCALE[i] - 1) + xy_grid) * STRIDES[i]
+        pred_xy = ((special.expit(conv_raw_dxdy) * XYSCALE[i]) - 0.5 * (XYSCALE[i] - 1) + xy_grid) * STRIDES[i]
```
Comment (vinitra) [inline vision/object_detection_segmentation/yolov4/README.md:103]: Is this import change necessary? It seems less readable than before.

Comment (vinitra) [reply vision/object_detection_segmentation/yolov4/README.md:103]: https://stackoverflow.com/questions/9819733/scipy-special-import-issue 
After looking into it, I see why this was necessary! Thank you for the fix.

**Comment by jennifererwangg** [comment]
Instead of calling `coco.names`, Could you add this [file](https://github.com/hunglc007/tensorflow-yolov4-tflite/blob/master/data/classes/coco.names) to the yolov4/dependencies folder and pass its file path to the `classes` argument of `draw_bbox()`?

In addition, will you make similar changes in the [inference.ipynb](https://github.com/onnx/models/blob/master/vision/object_detection_segmentation/yolov4/dependencies/inference.ipynb) notebook?

Thank you!

**Comment by mustafakasap** [comment]
> Can you make the changes from #350 in this PR itself?

Now two are merged. #350 is now under this PR.

**Comment by mustafakasap** [comment]
> Instead of calling` coco.names`, could you add this [file](https://github.com/hunglc007/tensorflow-yolov4-tflite/blob/master/data/classes/coco.names) to the yolov4/dependencies folder and pass its file path to the classes argument of `draw_bbox()`?
> 
> Thank you!

I am not sure from "Licence" perspective that we can host any external file here but just refer them. Otherwise sure it will be nice to have "coco.names" under dependencies.

**Comment by vinitra** [comment]
> > Instead of calling` coco.names`, could you add this [file](https://github.com/hunglc007/tensorflow-yolov4-tflite/blob/master/data/classes/coco.names) to the yolov4/dependencies folder and pass its file path to the classes argument of `draw_bbox()`?
> > Thank you!
> 
> I am not sure from "License" perspective that we can host any external file here but just refer them. Otherwise sure it will be nice to have "coco.names" under dependencies.

As the source repository is [licensed under the MIT license](https://github.com/hunglc007/tensorflow-yolov4-tflite/blob/master/LICENSE), and we give them credit for the original TF versions of the files, it should be fine to host a copy of the original coco categories in this repository. It's only 80 lines of category names. 

Once that's included, I'll approve and merge the PR! :)

**Comment by mustafakasap** [comment]
if you approve the license terms then sure I added the external file under dependencies folder now

**Pull request merged**